projects/arlas-components/src/lib/components/results/result-list/result-list.component.scss

```diff
@@ -164,6 +164,32 @@ tr {
 .resultlist__tools {
   height: 50px;
   overflow: hidden;
+
+  &--geosort /* @doc Geo-sort toggle in the list header */ {
```

Comment:
use `@doc` with sass notation will not generate the full class name in the documentation

Reply:
Shall I just use the full name then?